av_camera_launch/config/fsp_l_config.yaml

```diff
@@ -1,17 +1,25 @@
 /**:
   ros__parameters:
-    debug: True
-    compute_brightness: True
+    camera_name: "fsp_l"
+    camera_type: "chameleon"
+    serial_number: "17496155"
+    frame_id: "camera_fsp_l_optical"
+    parameter_file: "/opt/ros_ws/install/av_camera_launch/share/av_camera_launch/config/chameleon.yaml"
+    camerainfo_url: "file:///opt/ros_ws/install/av_camera_launch/share/av_camera_launch/config/fsp_l_calib.yaml"
+    debug: False
+    compute_brightness: False
     dump_node_map: False
     adjust_timestamp: True
     gain_auto: "Continuous"
     gain: 0
     exposure_auto: "Continuous"
     exposure_time: 9000
-    offset_x: 0
-    offset_y: 0
-    image_width: 2048
-    image_height: 1536
+    reverse_x: False
+    reverse_y: False
+    offset_x: 94
+    offset_y: 400
+    image_width: 2272
+    image_height: 1088
```

Comment:
Are these the parameters that need to be in an specific order so we avoid the bad-crop ROI issue when launching the cameras for the first time?